parsl/utils.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+import shlex
+
+import parsl
+from parsl.version import VERSION
+
+def get_version():
+    if 'site-packages' in __file__:
+        version = parsl.__version__
+    else:
+        start = os.getcwd()
+        os.chdir(os.path.dirname(__file__))
```

Comment:
I think this makes this `get_version` non-thread-safe as cwd is global - which is fine if nothing is happening when parsl is started up, but probably a surprise if anyone embeds it in something multithreaded that uses the filesystem.

If that's a valid concern, setting `GIT_DIR` in the subprocess environment or passing `--git-dir` as a parameter would be safer.